docs/get-started.md

```diff
@@ -0,0 +1,114 @@
+* [svcat CLI](https://svc-cat.io/docs/cli/) in version v1.10.0 or later
+
+## Steps
+
+### Create fork
+
+1. Create fork from the [bundle repository](https://github.com/kyma-project/bundles)
+
+2. On your forked repository create `gh-pages` branch if not exist
+
+3. Go to **Settings** tab and in section **GitHub Pages** setup **Source** to `gh-pages branch`
+
+### Integrate Travis-CI with Github repository
+
+1. Go to [Github](https://github.com) page.
+
+2. Choose **Settings** tab on your [Github profile account](https://github.com/settings/profile), in the left sidebar, click **Developer settings**.
+
+3. In the left sidebar, click **Personal access tokens**.
+
+4. Click **Generate new token**.
+
+5. Give your token a descriptive name.
+
+6. Select scope `repo -> public_repo` (Access public repository)
+
+7. Click **Generate token** and copy your personal access token.
+
+8. Go to [travis-ci](https://travis-ci.com/) and login using your Github account
+
+9. Add access to your forked `bundle` repository.
+
+10. In the left sidebar, click **My Repositories** tab and choose your Github repository
+
+11. In the right tab click **More options** then **Settings**
+
+12. Go to **Environment Variables** section and add key-value pair where key is `GITHUB_TOKEN` and value is equal to your Github personal access token.
+
+13. In the right tab click **More options** then **Trigger build** and wait for Travis to finish his work
+
+### Create bundle
+
+1. Create your bundle according to [instruction](https://kyma-project.io/docs/latest/components/service-brokers#configuration-how-to-create-a-ybundle)
+
+2. Place your bundle into the `bundle/{repository}` directory.
+
+3. Along your bundle directory create `index.yaml` which defines available bundles. 
+Instruction how to create a valid file can be found [here](https://github.com/kyma-project/kyma/blob/master/docs/service-brokers/docs/011-configuration-helm-broker.md#configuring-the-helm-broker-externally)
+
+4. Commit changes.
+
+5. Travis automatically launches new build. You can check build status on [travis-ci](https://travis-ci.com/) webpage.
+
+### Setup Kyma Helm Config Map
+
+1. Go to your forked repository, click **Settings** tab, then go to **GitHub Pages** section and copy Github pages link next to **Your site is published at** phrase
+
+2. Before you change `helm-config-map` base url to your repository url, go to terminal enter
+    ```
+    kubectl describe cm helm-config-map -n kyma-system | awk '/baseURL:/ {print $2}'
+    ```
+    and copy previous configuration.
+    
+3. Setup env variable on Helm Broker Deployment
+    ```
+    > kubectl set env -n kyma-system deployment/core-helm-broker -e APP_REPOSITORY_BASE_URL=https://user-name.github.io/bundles/{repository}/
+    ```
+    > **NOTE:** To be sure you have this config map you can display all config maps `kubectl get cm -n kyma-system`
+
+4. Wait until kubernetes restart `core-helm-broker` Pod.
```

Comment:
IMO we should be more precise
4. Wait until the Helm Broker Pods are in `READY` state:
```bash
watch -n 1 kubectl get pod -n kyma-system -l app=core-helm-broker
```

> **NOTE**: Before going to the next step, check the number of the Pods which should be in the `RUNNING` state by running this command:
>  ```bash
>  kubectl get deploy -n kyma-system core-helm-broker -ojsonpath='{ .spec.replicas }'
>  ```

but need to be also checked with Klaudia.

Reply:
I changed in your first sentence state from READY to RUNNING. READY could be confusing. Why do you assume users have "watch", it is not by default, it is easier to tell them use something which they already have ('kubectl' is in Prerequisites section).

Second command shows me "1%" and it tells me nothing, is it correct command?

Reply:
the command 
```
kubectl get deploy -n kyma-system core-helm-broker -ojsonpath='{ .spec.replicas }'
```
is definitely correct, so check your environment why you getting the additional `%` sign.

About: 
> I changed in your first sentence state from READY to RUNNING. READY could be confusing.

It's not exactly true in the case of Kuberntes. The READY and RUNNING are totally different. Let's talk about it f2f